website/style.css

```diff
@@ -337,6 +337,7 @@ pre.ttyshot {
 }
```

Comment:
It is possible they are no longer necessary. These CSS changes are inherited from my AsciiCinema pull-request that you rejected and formed the basis for this pull-request. Whether the "current color scheme is fine" is obviously a matter of opinion. 😸 I'll review whether the changes in question are needed or irrelevant.

Reply:
I reviewed the CSS changes. They fall into three categories. The first introduces changes to reflect changes I made to the SGR parsing code to distinguish between implicit (i.e., "default") and explicit SGR definitions; for example:

```
.sgr-39 { /* default foreground color */
    color: black;
}
.sgr-49 { /* default background color */
    background-color: white;
}
```

The second is to eliminate combinations of CSS selectors (e.g., `.sgr-42, .sgr-7.sgr-32`) because the new ttyshot generation mechanism doesn't require them and, AFAICT, nothing else required them.

The third aspect of this change is to replace color names like "gray" with an equivalent RGB hex value because I noticed that a couple of the color names had different RGB values depending on the browser and platform. I prefer the precision, and unambiguity, of a RGB hex value. This is the one aspect of my change to *website/style.css* that can be considered controversial. I tried to avoid making any significant changes to the rendered colors visible to a user. It's also the easiest to "undo" since I've included a comment for each new `sgr-` selector that makes it clear which "named" color it corresponds to.

Reply:
CSS 3 has standardized these color names since [Jun 2018](https://en.wikipedia.org/wiki/CSS#CSS_3), and the stylesheet uses CSS features that are more recent.

There is probably some merit in using hex values instead of color names, but the hex values you used are not those in the CSS. I presume you get them from your terminal emulator's profile.